types/react-stripe-elements/index.d.ts

```diff
@@ -45,8 +45,6 @@ export namespace ReactStripeElements {
 	interface ElementProps extends ElementsOptions {
 		className?: string;
 
-		paymentRequest?: object;
```

Comment:
Why have you removed this option? https://github.com/stripe/react-stripe-elements/blob/bf6a45fc036b7ec8821149d3def997a181ae8bf8/src/components/PaymentRequestButtonElement.js#L14

Reply:
I added the paymentRequest propery to the ElementOptions interface, so it's no longer necessary there (and it conflicts with the new defintion).

Reply:
👍